docs/configuration.md

```diff
@@ -0,0 +1,53 @@
+# Configuration
+
+## Configuration methods
+
+## Global settings
+
+| Environment variable        | Description                                                                | Default value |
+|-----------------------------|----------------------------------------------------------------------------|---------------|
+| `OTEL_GO_AUTO_TARGET_EXE`   | Sets the binary for the Go application to be instrumented.                 |               |
```

Comment:
`OTEL_GO_AUTO_TARGET_EXE` is a required environment variable (by default it is empty).